packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.ts

```diff
@@ -3,58 +3,72 @@ import { PrettySymbols } from 'phoenix-event-display';
 import { EventDisplayService } from '../../../../services/event-display.service';
 
 @Component({
-  selector: 'app-collections-info-overlay',
-  templateUrl: './collections-info-overlay.component.html',
-  styleUrls: ['./collections-info-overlay.component.scss']
+	selector: 'app-collections-info-overlay',
```

Comment:
For now, please configure your editor to use 2 spaces for tabs and format the TypeScript code. I intend to add prettier later but for now you will need to do this manually.

Reply:
Reverted and inserted the part of code into the original .ts file
Thanks a lot for the feedbacks!